opensearchpy/transport.py

```diff
@@ -197,13 +197,14 @@ def set_connections(self, hosts):
 
         :arg hosts: same as `__init__`
         """
+
         # construct the connections
         def _create_connection(host):
             # if this is not the initial setup look at the existing connection
             # options and identify connections that haven't changed and can be
             # kept around.
             if hasattr(self, "connection_pool"):
-                for (connection, old_host) in self.connection_pool.connection_opts:
+                for connection, old_host in self.connection_pool.connection_opts:
```

Comment:
Why make this change?

Reply:
linter is automatically formatting it. Without it lint error is produced.